drafts/2020-04-09-release-20004.adoc

```diff
@@ -0,0 +1,690 @@
+
+You can benefit from this dashboard while testing new apps to deploy on Open Liberty. For instance, metrics such as servlet response times, CPU or heap usage when seen as a time-series on Grafana, could be indicative of an underlying performance issue or memory leak.
+
+To configure the dashboard, first add the `mpMetrics-2.3` feature. This would automatically enable the `monitor-1.0` feature: 
+
+[source]
+----
+<featureManager>
+    <feature>mpMetrics-2.3</feature>
+</featureManager>
+
+<mpMetrics authentication="false" />`
+----
+
+For metrics on a secure endpoint:
+[source]
+----
+<featureManager>
+    <feature>mpMetrics-2.3</feature>
+</featureManager>
+
+<quickStartSecurity userName="<your-username>" userPassword="<your-password>" />
+----
+
+Run the server using the following command: 
+
+[source]
+----
+./server run DashboardTest
+----
+
+Then, download https://prometheus.io/download/[Prometheus]. Once unpackaged, it should contain a startup script called prometheus alongside a configuration file, `prometheus.yml`. Within `prometheus.yml`, append to `scrape_configs` one of the following jobs/
+
+For `mpMetrics` on an insecure endpoint:
+[source]
+----
+- job_name: 'liberty'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9080']
+----
+
+For `mpMetrics` on a secure endpoint:
+[source]
+----
+- job_name: 'liberty-secure'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9443']
+    basic_auth:
+      username: "<your-username>"
+      password: "<your-password>"
+    tls_config:
+       insecure_skip_verify: true
+    scheme: "https"
+----
+
+Start the prometheus script and visit http://localhost:9090/targets, where you should see your Open Liberty server listed as one of the targets.
+
+image::img/blog/20004-prometheus.png[align="center"]
+
+Download https://grafana.com/docs/grafana/latest/installation/[Grafana] and once finished, visit https://localhost:3000.
+
+When navigated to Grafana, click the gear icon and select "Data Source". Add a new Prometheus datasource with the URL as `\http://localhost:9090`. Then click "Save & Test".
+
+To import the dashboard, click the plus icon, select "Import", paste the dashboard ID 11706, and when prompted in the dropdown menu, link it to the new data source you have just created.
+
+image::img/blog/20004-grafana.png[align="center"]
+
+For more informtation: 
+
+* https://grafana.com/grafana/dashboards/11706[The dashboard found on Grafana's website]
+* https://prometheus.io/docs/prometheus/latest/querying/basics/[Using Prometheus to create your own custom visualizations]
+
+[#os]
+=== Support OpenShift service account credentials for authentication
+
+This new feature for Open Liberty means that application developers can use the new `socialLogin-1.0` feature allows applications to be secured by using popular social media OAuth and OpenIDConnect providers, and supports configuration for additional providers.
```

Comment:
change "feature" to "capability" (or something). I don't think it's a "feature" in the Liberty feature sense, so it's potentially confusing.